Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Add_IPs_to_Static_Address_Group.yml

```diff
@@ -0,0 +1,587 @@
+      name: Update address group
+      description: Update an existing address group.
+      script: '|||prisma-sase-address-group-update'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      folder:
+        complex:
+          root: inputs.Folder
+      group_id:
+        complex:
+          root: PrismaSase.AddressGroup
+          accessor: id
+      static_addresses:
+        complex:
+          root: inputs.IP
+          transformers:
+          - operator: uniq
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 56aafb2d-6019-47c4-8be8-2ccd2e3e72c4
+    type: regular
+    task:
+      id: 56aafb2d-6019-47c4-8be8-2ccd2e3e72c4
+      version: -1
+      name: Push Config
+      description: Push the candidate configuration.
+      script: '|||prisma-sase-candidate-config-push'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      folders:
+        simple: Remote Networks, Mobile Users, Service Connections
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 1270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: da1f402d-606d-4448-8805-0ee35ad05444
+    type: playbook
+    task:
+      id: da1f402d-606d-4448-8805-0ee35ad05444
+      version: -1
+      name: Prisma SASE - Create Address Object
+      description: This playbook creates new address objects in the Prisma SASE Object section. Those objects can be used later on in other objects such as Security Rules.
+      playbookName: Prisma SASE - Create Address Object
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      AutoCommit:
+        simple: Else
+      Folder:
+        complex:
+          root: inputs.Folder
+      IP:
+        complex:
+          root: inputs.IP
+          filters:
+          - - operator: notIn
+              left:
+                value:
+                  simple: inputs.IP
+                iscontext: true
+              right:
+                value:
+                  simple: PrismaSase.AddressGroup.addresses
+                iscontext: true
+          transformers:
+          - operator: uniq
+      TSGID:
```

Comment:
I dont see the 'type' input of the subplaybook. In other playbook you added the value 'ip_netmask' to the type input. Is it empty in purpose?

Reply:
Indeed it should be specified - added `Type` to be "ip_netmask". Great catch!